HazelAPI/src/main/java/net/juligames/core/api/hazel/NativeHazelDataAPI.java

```diff
@@ -58,4 +62,8 @@ public interface NativeHazelDataAPI {
      * @return the ITopic associated to the hazel
      */
     <T> ITopic<T> getTopic(String hazel);
+
+    @ApiStatus.Experimental
+    @ApiStatus.AvailableSince("1.5")
+    Collection<DistributedObject> getAll();
```

Comment:
This might present the same issue as the DistributedObjectInterpreter. Further investigation is needed!

Reply:
I will investigate this further after the release of 1.5! I don’t think this will cause issues and it is also very important to get this version and its associated features